model_analyzer/record/metrics_manager.py

```diff
@@ -160,6 +153,23 @@ def profile_model(self, run_config, perf_output_writer=None):
         cpu_only = run_config.model_config().cpu_only()
         perf_config = run_config.perf_config()
 
+        # Inform user CPU metric(s) are not being collected under CPU mode
+        collect_cpu_metrics_expect = cpu_only or len(self._gpus) == 0
+        collect_cpu_metrics_actual = len(self._cpu_metrics) > 0
+        if collect_cpu_metrics_expect and not collect_cpu_metrics_actual:
+            logging.info(
+                "CPU metric(s) are not being collected, while this profiling will run on CPU(s)."
+            )
+        # Warn user about CPU monitor performance issue
+        if collect_cpu_metrics_actual:
+            logging.warning("CPU metric(s) are being collected.")
+            logging.warning(
+                "Collecting CPU metric(s) during profiling might affect performance!"
+            )
+            logging.info(
+                "CPU metric(s) collection can be disabled by changing the `--metrics` flag."
```

Comment:
"CPU metric(s) collection can be disabled by removing the CPU metrics (e.g. cpu_used_ram) from the `--metrics` flag."

Reply:
Updated.